src/plugins/intel_cpu/src/nodes/executors/acl/acl_fullyconnected.cpp

```diff
@@ -0,0 +1,297 @@
+                                     const ACLFCAttrs& aclfcAttrs,
+                                     const PostOps &postOps) {
+    DEBUG_LOG("ACLFullyConnectedExecutor: prepack weights");
+    const auto& wgtDims = memory.at(ARG_WEI)->getStaticDims();
+    const auto N = wgtDims[0];
+    const auto K = wgtDims[1];
+
+    auto create = [&]() {
+        MemoryPtr final_ptr = memory.at(ARG_WEI);
+        if (aclfcAttrs.isConvertedWeights) {
+            MemoryArgs memoryArgs;
+            memoryArgs[ARG_SRC_0] = memory.at(ARG_WEI);
+            memoryArgs[ARG_DST] = std::make_shared<Memory>(context->getEngine(),
+                                                           memoryArgs[ARG_SRC_0]->getDescPtr()->cloneWithNewPrecision(
+                                                                   aclfcAttrs.inputPrecision));
+            auto aclWeightsConverter = std::make_shared<acl_fc_executor::ACLWeightsConverter>();
+            if (aclWeightsConverter->update(memoryArgs)) {
+                aclWeightsConverter->execute(memoryArgs);
+            } else {
+                auto count_wei_elem = std::accumulate(memoryArgs[ARG_SRC_0]->getStaticDims().begin(),
+                                                      memoryArgs[ARG_SRC_0]->getStaticDims().end(),
+                                                      1,
+                                                      std::multiplies<>());
+                cpu_convert(memoryArgs[ARG_SRC_0]->getData(),
+                            memoryArgs[ARG_DST]->getData(),
+                            memoryArgs[ARG_SRC_0]->getPrecision(),
+                            memoryArgs[ARG_DST]->getPrecision(),
+                            count_wei_elem);
+            }
+            final_ptr = memoryArgs[ARG_DST];
+        }
+        if (!aclfcAttrs.weightsNonTransposed) {
+            auto reverse_weights_dims = memory.at(ARG_WEI)->getStaticDims();
+            if (reverse_weights_dims.size() == 3) {
+                reverse_weights_dims = VectorDims(
+                        {reverse_weights_dims[0] * reverse_weights_dims[1], reverse_weights_dims[2]});
+            }
+            std::reverse(reverse_weights_dims.begin(), reverse_weights_dims.end());
+            MemoryArgs memoryArgs;
+            memoryArgs[ARG_SRC_0] = final_ptr;
+            memoryArgs[ARG_DST] = std::make_shared<Memory>(context->getEngine(),
+                                                           CpuBlockedMemoryDesc(final_ptr->getPrecision(),
+                                                                                intel_cpu::Shape(reverse_weights_dims)));
+            auto aclWeightsTranspose = std::make_shared<acl_fc_executor::ACLWeightsTranspose>();
+            if (aclWeightsTranspose->update(memoryArgs)) {
+                aclWeightsTranspose->execute(memoryArgs);
+                final_ptr = memoryArgs[ARG_DST];
+            }
+        }
+        if (!memory.at(ARG_SRC)->getShape().isDynamic()) {
```

Comment:
@EgorDuplensky  @allnes  Is it possible SRC to be dynamic in this context?

Reply:
We need to use dummyshapes here, similar to oneDNN executor

Reply:
@dmitry-gorokhov added dummyShapes function